database.sql

```diff
@@ -1,6 +1,6 @@
 -- ------------------------------------------
 -- Friendica 2022.12-rc (Giant Rhubarb)
--- DB_UPDATE_VERSION 1501
+-- DB_UPDATE_VERSION 1502
```

Comment:
why? :)

Reply:
To make sure the `update_1502` method is executed. I don't know if a `update_1501` method would be executed if the current DB version already is 1501, so I didn't take any chances.